src/test/unit/profile-data.test.js

```diff
@@ -1028,3 +1030,174 @@ describe('convertStackToCallNodePath', function() {
     expect(callNodePath).toEqual([5, 3, 2, 1, 0]);
   });
 });
+
+describe('getTimingsForPath in a non-inverted tree', function() {
+  function setup() {
+    const { profile, funcNamesDictPerThread } = getProfileFromTextSamples(`
+      A    A    A A
+      B    B    B B
+      Cjs  Cjs  H H
```

Comment:
Other existing tests use `C.js` instead of `Cjs`. Not sure if it's worth being consistent with those here.

Reply:
Oh, I see, using `Cjs` allows you to use the `funcNamesDict` more effectively. Let's keep it that way, then.

Reply:
Yeah. I thought of modifying the getProfileFromTextSamples to provide the func names without the extension; but then I thought it was just good enough to do it like this...